src/main/java/com/ldtteam/structurize/blocks/schematic/BlockSubstitution.java

```diff
@@ -40,4 +42,15 @@ public VoxelShape getShape(final BlockState state, final BlockGetter worldIn, fi
     {
         return Shapes.box(.125D, .125D, .125D, .875D, .875D, .875D);
     }
+
+    @Override
+    public VoxelShape getCollisionShape(BlockState state, BlockGetter blockGetter, BlockPos blockPos, CollisionContext context) {
+        // Allow players to move through placeholders with a scantool in hand
+        if (context.isHoldingItem(ModItems.scanTool.get()))
```

Comment:
This only checks the item in the active hand; it might be nicer to check the offhand slot too (or even better, the whole hotbar).  Otherwise builders will keep getting stuck in the middle of placeholders when placing other blocks and have to keep swapping back.

Reply:
I kinda wanted to have it easily swappable, but offhand is a good idea, since it allows for both